loggo/loggo.py

```diff
@@ -77,58 +47,37 @@ class Loggo(object):
     - do_print: print logs to console
     - do_write: write logs to file
     - line_length: max length for console printed string
-    - private_data: key names that should be filtered out of logging. if not set,
+    - private_data: key names that should be filtered out of logging
       some sensible defaults are used
     """
     def __init__(self, config={}):
         self.stopped = False
         self.allow_errors = True
         self.config = config
         self.sublogger = None
-        self.nargs = 0
-        self.nkwargs = 0
+        # these things should always end up in the extra data provided to logger
         self.log_data = dict(loggo=True, loggo_config=dict(config), sublogger=self.sublogger)
         self.facility = config.get('facility', 'loggo')
-        self.do_colour = config.get('colour', False)
         self.ip = config.get('ip', None)
         self.port = config.get('port', None)
         self.do_print = config.get('do_print', False)
         self.do_write = config.get('do_write', False)
         self.logfile = config.get('logfile', './logs/logs.txt')
         self.line_length = config.get('line_length', 200)
         self.obscured = config.get('obscure', '[PRIVATE_DATA]')
-        self.private_data = config.get('private_data', DEFAULT_PRIVATE_KEYS)
+        self.private_data = config.get('private_data')
```

Comment:
Would be great to have consistency here with `self.ip = config.get('ip', None)` et al above. I think it's pretty clear to everyone that `None` is default so I'd prefer omitting it like on this line